packages/ag-charts-community/src/module/optionsModule.ts

```diff
@@ -256,6 +257,8 @@ export class ChartOptions<T extends AgChartOptions = AgChartOptions> {
         // so we aim to only do this once in the processing flow.
         processedOptions = deepClone(processedOptions, ChartOptions.OPTIONS_CLONE_OPTS);
 
+        this.processOptionsParams(processedOptions);
```

Comment:
Since this visits every node within the object, it may be possible to combine it with the cloning and only do a single iteration.

But that may depend on how we handle ref's that point to ref's (if we don't just throw an error on that).